modules/ROOT/pages/getting-started.adoc

```diff
@@ -82,7 +92,7 @@ Most (but not all) RPM packages provided by Fedora can be installed on {variant-
 
 Currently, using package layering creates a new "deployment", or bootable filesystem root.
 It does not affect your current root.
-This preserves rollback and the transactional model, but means that the system must be rebooted after a package has been layered.
+This preserves rollback and the transactional model, but means that the system must be rebooted after a package has been layered or updated.
```

Comment:
Can you add a mention to `rpm-ostree install --apply-live <pkg>`? Thanks!

Reply:
Wasn't that an experimental feature?

Reply:
This one is stable now: https://github.com/coreos/rpm-ostree/releases/tag/v2022.13